fictadvisor-api/src/v2/utils/documentation/pageText/getAllTexts.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiDocumentationParams } from '../decorators';
```

Comment:
Name of the documentation file (getAllTexts.ts) must be the same as corresponding controller method (getAll.ts)